install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -188,19 +188,20 @@ type IAMConfig struct {
 }
 
 type WebAppConfig struct {
-	PublicAPI              *PublicAPIConfig       `json:"publicApi,omitempty"`
-	Server                 *ServerConfig          `json:"server,omitempty"`
-	ProxyConfig            *ProxyConfig           `json:"proxy,omitempty"`
-	WorkspaceManagerBridge *WsManagerBridgeConfig `json:"wsManagerBridge,omitempty"`
-	Tracing                *Tracing               `json:"tracing,omitempty"`
-	UsePodAntiAffinity     bool                   `json:"usePodAntiAffinity"`
-	DisableMigration       bool                   `json:"disableMigration"`
-	Usage                  *UsageConfig           `json:"usage,omitempty"`
-	ConfigcatKey           string                 `json:"configcatKey"`
-	WorkspaceClasses       []WebAppWorkspaceClass `json:"workspaceClasses"`
-	Stripe                 *StripeConfig          `json:"stripe,omitempty"`
-	SlowDatabase           bool                   `json:"slowDatabase,omitempty"`
-	IAM                    *IAMConfig             `json:"iam,omitempty"`
+	PublicAPI                  *PublicAPIConfig       `json:"publicApi,omitempty"`
+	Server                     *ServerConfig          `json:"server,omitempty"`
+	ProxyConfig                *ProxyConfig           `json:"proxy,omitempty"`
+	WorkspaceManagerBridge     *WsManagerBridgeConfig `json:"wsManagerBridge,omitempty"`
+	Tracing                    *Tracing               `json:"tracing,omitempty"`
+	UsePodAntiAffinity         bool                   `json:"usePodAntiAffinity"`
+	DisableMigration           bool                   `json:"disableMigration"`
+	Usage                      *UsageConfig           `json:"usage,omitempty"`
+	ConfigcatKey               string                 `json:"configcatKey"`
+	WorkspaceClasses           []WebAppWorkspaceClass `json:"workspaceClasses"`
+	Stripe                     *StripeConfig          `json:"stripe,omitempty"`
+	SlowDatabase               bool                   `json:"slowDatabase,omitempty"`
+	IAM                        *IAMConfig             `json:"iam,omitempty"`
+	WithoutWorkspaceComponents bool                   `json:"withoutWorkspaceComponents,omitempty"`
```

Comment:
New experimental config field added: `WithoutWorkspaceComponents`.

Defaults to `false` (no change in behaviour). Setting to `true` will:
- Remove `ws-manager` TLS cert mounts in application clusters
- Prevent `ws-manager-bridge` from trying to connect to an in-cluster `ws-manager` (same as `skipSelf = true`)
- Prevent `ws-proxy` in application cluster from trying to connect to an in-cluster `ws-manager`
- Force `server` to talk to `image-builder-mk3` in workspace clusters, instead of in the application cluster.